docs/disaster-recovery.md

```diff
@@ -0,0 +1,443 @@
+```
+
+> ```bash
+> customresourcedefinition.apiextensions.k8s.io/volumereplications.replication.storage.openshift.io created created
+> ```
+
+The VolumeReplicationClass and VolumeReplication CRDs are now created.
+
+>:bulb: **Note:** Use the latest available release for Volume Replication Operator.
+> See [releases](https://github.com/csi-addons/volume-replication-operator/branches)
+> for more information.
+
+### Add RBAC rules for Volume Replication Operator
+
+Add the below mentioned rules to `rbd-external-provisioner-runner`
+ ClusterRole in [csi-provisioner-rbac.yaml](https://github.com/ceph/ceph-csi/blob/release-v3.3/deploy/rbd/kubernetes/csi-provisioner-rbac.yaml)
+
+```yaml=61
+  - apiGroups: ["replication.storage.openshift.io"]
+    resources: ["volumereplications", "volumereplicationclasses"]
+    verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]
+  - apiGroups: ["replication.storage.openshift.io"]
+    resources: ["volumereplications/finalizers"]
+    verbs: ["update"]
+  - apiGroups: ["replication.storage.openshift.io"]
+    resources: ["volumereplications/status"]
+    verbs: ["get", "patch", "update"]
+  - apiGroups: ["replication.storage.openshift.io"]
+    resources: ["volumereplicationclasses/status"]
+    verbs: ["get"]
+```
+
+### Deploy the Volume Replication Sidecar
+
+Volume Replication Operator is a kubernetes operator that provides common
+ and reusable APIs for storage disaster recovery.
+ It is based on [csi-addons/spec](https://github.com/csi-addons/spec)
+ specification and can be used by any storage provider.
+
+To deploy `volume-replication` sidecar container in `csi-rbdplugin-provisioner`
+ pod, add the following yaml to
+ [csi-rbdplugin-provisioner deployment](https://github.com/ceph/ceph-csi/blob/release-v3.3/deploy/rbd/kubernetes/csi-rbdplugin-provisioner.yaml).
+
+```yaml=1
+        - name: volume-replication
+          image: quay.io/csiaddons/volumereplication-operator:v0.1.0
+          args :
+            - "--metrics-bind-address=0"
+            - "--leader-election-namespace={{ .Namespace }}"
+            - "--driver-name=rbd.csi.ceph.com"
+            - "--csi-address=$(ADDRESS)"
+            - "--rpc-timeout=150s"
+            - "--health-probe-bind-address=:9998"
+            - "--leader-elect=true"
+          env:
+            - name: ADDRESS
+              value: unix:///csi/csi-provisioner.sock
+          imagePullPolicy: "IfNotPresent"
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+```
+
+## VolumeReplicationClass and VolumeReplication
+
+### VolumeReplicationClass
+
+*VolumeReplicationClass* is a cluster scoped resource that contains
+ driver related configuration parameters. It holds the storage admin
+ information required for the volume replication operator.
+
+### VolumeReplication
+
+*VolumeReplication* is a namespaced resource that contains references
```

Comment:
Add note for PVC and VR objects need to be created on the same namespace.

Reply:
Mentioned below it https://github.com/ceph/ceph-csi/pull/2320/files#diff-e5eca4c1c003e7e66c84308840990e93ee14553d6643533a5a54a106a91387f7R157